haddock-library/fixtures/examples/table-simple.parsed

```diff
@@ -3,50 +3,44 @@ DocTable
     {tableBodyRows = [TableRow
                         [TableCell
                            {tableCellColspan = 1,
-                            tableCellContents = DocString " 200  ",
+                            tableCellContents = DocString " 200",
                             tableCellRowspan = 1},
                          TableCell
                            {tableCellColspan = 1,
                             tableCellContents = DocAppend
                                                   (DocString "   ")
-                                                  (DocAppend
-                                                     (DocMonospaced (DocString "OK"))
-                                                     (DocString "       ")),
+                                                  (DocMonospaced (DocString "OK")),
                             tableCellRowspan = 1},
                          TableCell
                            {tableCellColspan = 1,
-                            tableCellContents = DocString
-                                                  " operation successful                     ",
+                            tableCellContents = DocString " operation successful",
```

Comment:
I wonder if we should strip leading whitespace as well?

Reply:
I will look into that as a additional task.

Reply:
It involved a trivial change but I have added code that strips leading whitespace as well. Also test fixtures and one html-test reference have been updated for this.